### PR TITLE
Attachments: attach images to posts

### PR DESCRIPTION
**Pull request by clbn**

### CODE REVIEW

File: public/js/app/fields/CreateAttachmentView.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        var newFile = event.target.files[0]
+        this.get('controller').send('addAttachment', newFile)
+
+        // Clear the file input in DOM: 1) wrap with the form, 2) reset the form, 3) unwrap
+        this.$().wrap('<form>').closest('form').get(0).reset()
+        this.$().unwrap()
+      }
+    },
+
+    click: function() {
```
Comment (epicmonkey) [inline public/js/app/fields/CreateAttachmentView.js]: Do we really need this action? I mean FrF does not show send-to field when you click on Add Photos/Files link.

**Comment by epicmonkey** [comment]
Pretty clean, like this.

**Comment by epicmonkey** [comment]
There are two more controllers that can submit post (and so they have create new post action):
* TimelineIndexController
* TimelineDiscussionsController

Let's create a generic controller that TimelineIndex, TimelineHome and TimelineDiscussions will inherit from.

File: public/js/app/fields/CreateAttachmentView.js (hunk 1 above)
Comment (clbn) [inline public/js/app/fields/CreateAttachmentView.js]: You are right, I wasn't aware of this. I think the question is, whether it's a feature we'd like to keep, or a bug we'd like to fix. :)

**Comment by clbn** [comment]
I agree about controllers. I will add a generic one in another commit to this branch tomorrow.

File: public/js/app/fields/CreateAttachmentView.js (hunk 1 above)
Comment (epicmonkey) [inline public/js/app/fields/CreateAttachmentView.js]: FrF is our cargo-cult. For the time being we won't be doing anything new but cloning/restoring that FrF had.

**Comment by clbn** [comment]
I've tried to add Submit Post functionality to other timelines (via generic controller), but there is at least one side effect (list of attached images "leaks out" to the other timeline's post form).

I would suggest merging this one and then isolating TimelineGenericController as a separate task since it needs some additional work.

**Pull request merged**